guide/daily-spending-wallet/settings.md

```diff
@@ -0,0 +1,213 @@
+    - file: lightning-network
+      modalImage: lightning-network-big
+      alt: Mobile screen with lightning network customization options
+      caption: Information about the users node, and additional security options.
+    - file: watchtower
+      modalImage: watchtower-big
+      alt: Mobile screen with lightning network watchtower options
+      caption: Users can enable watchtower functionality to protect their funds.
+---
+
+{% include picture.html
+    image = "/assets/images/guide/daily-spending-wallet/settings/settings.jpg"
+    retina = "/assets/images/guide/daily-spending-wallet/settings/settings@2x.jpg"
+    mobile = "/assets/images/guide/daily-spending-wallet/settings/settings-mobile.jpg"
+    mobileRetina = "/assets/images/guide/daily-spending-wallet/settings/settings-mobile@2x.jpg"
+    alt-text = "An oil painting of a big, clunky machine with a big bitcoin gear"
+    caption = "How AI thinks Matisse would have visualized bitcoin settings."
+    width = 1600
+    height = 600
+    layout = "full-width"
+%}
+
+<!--
+
+
+This is more of a summary page that bring together all the application settings that
+the user encounters throughout the app experience into a single, concise form.
+
+
+-->
+
+# Settings
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/settings/overview.png"
+   retina = "/assets/images/guide/daily-spending-wallet/settings/overview@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/settings/overview-big.png"
+   alt-text = ""
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+   caption = "The overview screen provides an easy-to-understand grouping of settings."
+%}
+
+As you probably noticed in previous pages, various settings can be configured contextually throughout the app experience (like [hiding and unhiding the balance]({{ '/guide/daily-spending-wallet/privacy/#hiding-sensitive-information' | relative_url }})). All these options, and a few extra ones, are bundled in the settings area.
+
+While not the most frequently visited part of an application, it fills an important functional role by giving users precise control over their experience.
+
+</div>
+
+## General
+
+This sub-section includes various non-essential settings that are not covered in the other categories and let users adjust the application to their context and personal preferences.
+
+{% include image-gallery.html pages = page.images_general %}
+
+## Fees
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/settings/fees.png"
+   retina = "/assets/images/guide/daily-spending-wallet/settings/fees@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/settings/fees-big.png"
```

Comment:
Notifications option isn't cantered properly in first screen

What does incoming transactions toggle mean? No context to it.

Reply:
The "All options" looks like it could be selected in the second image when its a title.